### DISCUSSION TITLE
More intuitive text to custom variable placeholder list

### DISCUSSION THREAD

**Discussion by marcosalpereira** [grafana/grafana#48816] [Dashboards, Panels & Visualizations]
I guess it will be a more intuitive placeholder attribute at "Variables > Edit" form

From:
`1, 10, mykey : myvalue, myvalue, escaped\,value`

To:
`1, 10, text : value, myvalue, escaped\,value`

![image](https://user-images.githubusercontent.com/3206979/167140774-f0e4ae15-3e9e-4a72-a9f8-e79a9a63713e.png)

ps: these `spaces` before and after `:` are really necessary?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
